Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/LightWeightExtensionSameplePage.xaml

```diff
@@ -0,0 +1,508 @@
+﻿<Page x:Class="Uno.Gallery.Views.SamplePages.LightWeightExtensionSameplePage"
+	  xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	  xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	  xmlns:local="using:Uno.Gallery"
+	  xmlns:smtx="using:ShowMeTheXAML"
+	  xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+	  xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	  mc:Ignorable="d"
+	  Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+
+	<Grid>
+		<local:SamplePageLayout>
+			<local:SamplePageLayout.MaterialTemplate>
+				<DataTemplate>
+					<StackPanel>
+						<!-- Button -->
+						<smtx:XamlDisplay UniqueKey="LightweightExtensionSamplePage_Button_Material"
+										  smtx:XamlDisplayExtensions.Description="The examples contrast ordinary Buttons with others that apply an Extension to override some properties, as defined in the Material/Buttons.xaml ResourceDictionary."
+										  smtx:XamlDisplayExtensions.IgnorePath="XamlDisplay\StackPanel">
+							<StackPanel Spacing="40">
+
+								<Button Content="Default Button Style"
+										Style="{StaticResource MaterialFilledButtonStyle}" />
+
+								<Button Content="Overridden Button Style"
+										BorderThickness="2"
+										Style="{StaticResource OverridenMaterialButton}" />
+
+								<!-- OverridenMaterialButton defintion: -->
+								<!--<Style x:Key="OverridenMaterialButton"
```

Comment:
dont just copy paste the referenced sources
format them accordingly to the indent level

Reply:
If you're talking about the comments, I did that for the show me the xaml. It looks weird if you put tabs in the comments.

Reply:
![image](https://github.com/unoplatform/Uno.Gallery/assets/90481654/5a69d02a-43f6-4ca3-837f-5a3592ac9da7)

Reply:
I removed the tabs because I prefer that it looks like the left vs the right. I could also make each individual line a comment instead of comments block, that should also work.

Reply:
II guess I could also completely remove the commented lines